eng/scripts/typespec/Build-Codegen.ps1

```diff
@@ -11,7 +11,7 @@ param(
 npm install -g @microsoft/rush typescript autorest @typespec/compiler @azure-tools/typespec-client-generator-cli @azure-tools/typespec-azure-rulesets
 
 pushd
-cd ..\..\
+cd ..\..\..\..\
```

Comment:
See [other comment](https://github.com/Azure/azure-sdk-for-cpp/pull/6463/files?file-filters%5B%5D=.cmake&file-filters%5B%5D=.cpp&file-filters%5B%5D=.hpp&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.ps1&file-filters%5B%5D=.tsp&file-filters%5B%5D=.txt&file-filters%5B%5D=.yaml&file-filters%5B%5D=.yml&owned-by%5B%5D=danieljurek&show-deleted-files=true&show-viewed-files=true#r1996177861) about `$PSScriptRoot`

Reply:
same as before